app/views/challenge/shared/_challenge_form_confirm_delete.html.erb

```diff
@@ -0,0 +1,14 @@
+<%= form_for(@collection, :url => 
+  if @collection.prompt_meme?
+    collection_prompt_meme_path(@collection)
+  else
+    collection_gift_exchange_path(@collection)
+  end,
+  :html => {:method => :delete}) do |f| %>
+  <p class="caution notice">
+    <%= ts("Are you certain you want to <strong><em>delete</em></strong> the challenge from this collection? All sign-ups, assignments, and settings will be lost. (Works will remain in the collection).").html_safe %>
```

Comment:
Hmm. This text is different from what's in the issue, but I see why: you went with text that matches the existing confirmation text. That makes sense! But it could use a few tweaks. 

- We're currently inconsistent about this throughout the code, but let's go with "Are you sure" over "certain," if only because we use "sure" in [the fallback text](https://github.com/otwcode/otwarchive/blob/17307a8a6294d2af7fab882ea5aa14eb786d51af/public/javascripts/application.js#L421) as well.
- Instead of "This collection," we should still reference the collection name. We generally try to do that when you're taken to a separate deletion page, just to make it a bit clearer what you're deleting.
- It should say "Works and bookmarks will remain in the collection," and the period should be inside the parentheses.